pkg/apis/core/validation/validation_test.go

```diff
@@ -6686,6 +6686,33 @@ func TestValidatePod(t *testing.T) {
 				DNSPolicy:     core.DNSClusterFirst,
 			},
 		},
+		{ // valid serviceaccount token projected volume with serviceaccount name specified
+			ObjectMeta: metav1.ObjectMeta{Name: "valid-extended", Namespace: "ns"},
```

Comment:
Just a query that , Why we are not specifying expectedError: here.?

Reply:
Because we don't expect an error.

Reply:
no error is expected